features/sync.json

```diff
@@ -3,5 +3,20 @@
         "description": "Sync & Backup browser feature. Feature available on native clients. Allow users to sync bookmarks, logins, etc.",
         "sampleExcludeRecords": {}
     },
+    "state": "internal",
```

Comment:
Hate to be a pain but we should probably default disable this on all platforms and just set it to internal where needed. We don't want any unintended side effects on other platforms even if they don't use the feature.

Reply:
I wonder about that. Is 'internal' a state that is expected to be globally supported? If not, are there plans for that? Cause what I'd expect it to be able to use internal state exactly in situations like this - where we are developing something and we have it as an internal feature across products (for products that don't support given feature it should not matter - as they won't parse the flag in the first place).

Reply:
In theory it should work like @bwaresiak said, but I also prefer to err on the side of being careful here and leave default state as `disabled`.  At least is consistent with the value when the state is not present.